software/spmd/quicksort/main.c

```diff
@@ -119,7 +119,7 @@ void quicksort(int* lst, int n)
   }
   else
   {
-    bsg_fail_x(3);
+    bsg_fail_x(0);
```

Comment:
Can you call bsg_fail instead? That doesn't depend on the hard-coded x coordinate